addons/im_livechat/static/src/core/web/thread_patch.js

```diff
@@ -0,0 +1,45 @@
+import { Thread } from "@mail/core/common/thread";
+import { _t } from "@web/core/l10n/translation";
+import { user } from "@web/core/user";
+import { patch } from "@web/core/utils/patch";
+
+const { DateTime } = luxon;
+
+patch(Thread.prototype, {
+    get livechatVisitor() {
+        return this.props.thread.livechatVisitorMember;
+    },
+    get showVisitorDisconnected() {
+        return (
+            this.props.thread.channel_type === "livechat" &&
+            this.store.self.isInternalUser &&
+            this.props.thread.livechat_active &&
+            this.livechatVisitor &&
+            this.livechatVisitor.persona.im_status &&
+            !this.livechatVisitor.persona.im_status.endsWith("online")
```

Comment:
Do we really want to display "disconnected" when the visitor is away?
I think it would be better to target "includes offline" specifically?